src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
```

Comment:
```suggestion
@Table(name = "user")
```

Reply:
Don't agree. My table expected for multiple users, not for 1.